README.md

```diff
@@ -1,2 +1,26 @@
-# base-app
-A simple JS app.
+# Awesome Tasks
+
+A simple JavaScript app that will help you getting things done.
+
+## Key concepts
+
+In order to understand the code of this simple app, is necessary to grasp these concepts:
+
+### JavaScript
+  * Base values: `String`, `Integer`, `Boolean`, undefined values
+  * Variables and functions
+  * Compound values: `Array` and `Object`
+  * Language Structure: `if`, ternary operator
```

Comment:
Not sure if we want to introduce the ternary operator, I think giving a sense of the possibility to direct the program flow given a condition is enough to begin with.